<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeManagementDialog</class>
 <widget class="QDialog" name="EmployeeManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="statusLineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addEmployeeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeEmployeeButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee List</string>
   </property>
  </widget>
  <widget class="QListWidget" name="employeeListWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
